src/main/java/org/usfirst/frc/team1758/robot/subsystems/DriveTrain.java

```diff
@@ -3,46 +3,27 @@
 import org.usfirst.frc.team1758.robot.RobotMap;
 import org.usfirst.frc.team1758.robot.commands.DriveWithJoystick;
 
+import com.ctre.CANTalon;
+
+
```

Comment:
Try and keep multiple spaces out of the project. Single space is fine but more than that is a lot of extra white space.